### ISSUE TITLE
the wiki sitemap is not linked anywhere

### ISSUE THREAD

**Issue by yugecin** [ppy/osu-wiki#1652]
The sitemap (https://osu.ppy.sh/help/wiki/Sitemap/) seems to be only linked from `Main_Page`, but that page does not exist on the live version.

**Comment by MegaApplePi** [comment]
The sitemap was intended to be an orphan, though I suppose it can be linked in the `Main_Page` (when it's done) and the `Welcome` page. There are a few redirects that will go there, `all_articles`, `all_pages`, `all_links`, and `index`.

**Comment by yugecin** [comment]
I find it weird that it's intended to be an orphan because how are users supposed to find out that there is in fact a sitemap page available if it's not linked anywhere? I wouldn't exactly expect people to stumble upon it when they randomly try any of the mentioned redirects.

It seems useful to me if it's referenced somewhere (maybe on every page at the bottom?) for when one gets lost in the wiki or is trying to find a specific page but doesn't know how to get there (I experienced this many times before).

Since it is auto generated, it may also help with finding pages that are not linked anywhere. For example, the shorthand page was only linked in the sitemap and I had no idea it existed until I did some digging in this repository (PR #1661 should fix this specific case).
